content/news/2024-12-12-mixxing-up-pen-and-paper.md

```diff
@@ -0,0 +1,122 @@
+>
+> It hums. A small melody. Fighting against the eary noise around you.
+
+If I had to pick a favorite moment in a video game, it would be this: a perfect example of how mixing and dynamic soundtracks can not only enhance, but tell a story outright.
+So let's use this to spice up pen and paper.
+
+*[Play "Love as a construct" from Portal 2 on YouTube Music.](https://music.youtube.com/watch?v=UNhBzbN2rnA)*
+
+## Why use Mixxx for Pen and Paper
+
+There are many different flavors of Tabletop Role Playing Games like Dungeons and Dragons, Vampires the Masquerade or Fate. And there are many ways to play it. Some like it silly, some like it tactical, and I personally like it intense and immersive.
+And Music is great for this.
+
+Most Storytellers will just use one of the many playlists offered by youtube or one of the many online tabletop audio apps. But that never really worked for me.
+
+One obvious reason is: Internet on Pen and Paper Cons is often bad.
+
+But I also like running campaigns in existing worlds. Alan Wake, Halo, Baldur's Gate, Darksiders, Dark Souls - many games with very memorable soundtracks.
+
+So just use these Soundtracks for my campaigns?
+
+Well that is easier said than done.
+
+*[Play "Never split the Party"](https://www.youtube.com/watch?v=k6y4XYxhA-o)*
+
+## How does a video game soundtrack work?
+
+Video game soundtracks are weird pieces of music, because play is unpredictable. Players will take different amounts of time to go through each level, so a Soundtrack has to adapt to this.
+
+This can lead to the pieces of music that are actually being used in a videogame being quite specific, quite fragmented and not useful for listening on its own. For a release on spotify or as bonus content, the original game sound files are being merged into pleasant songs, sometimes with more, sometimes with less effort. And this is what I have to work with.
+
+And of course you can just take these and put them on loop in VLC. But especially modern game soundtracks will vary widely in theme within a single song, so you constantly switch between motions on a looping timer and then there are also intros and outros to add further annoyances.
+
+But we have Mixxx and with the newly-added loop cues, we can work with almost every quirk game soundtracks can throw at us.
+
+### Cinematic Songs
+Most game soundtracks will have some songs made for specific cinematics or cut-scenes. They generally get sorted into the "overly specific" crate for me.
+
+That is because of the strong thematic flow and sometimes even tempo shifts, that make them really difficult to mix for me.	
+
+However: when planning a scene to follow the beat of a song, these  will create some really spectacular moments. You just have to practice your timing and be a very confident Dungeon Master if you want to allow player interaction in the fixed time frame of a track.
+
+*[Watch "Hyper Light Drifter - Opening"](https://www.youtube.com/watch?v=9rVdUufUBss)*
```

Comment:
Can you point the reader to sections in the track that demonstrates your statements?  Maybe do the same with the other linked tracks as well if there is a point of particular interest.

Reply:
Did you have trouble hearing out the features I highlighted when describing the different types of mixes?

For instance with this: the whole track just varies overall and is very specific to the visuals in the cut-scene (hence I liked a video)

Reply:
I must admit yes, since the link to the track appears without any explanation. 
When listening to the track I was not sure what exactly you mean.  

> For instance with this: the whole track just varies overall and is very specific to the visuals in the cut-scene (hence I liked a video)

Such a statement would be helpful.